x/subspaces/keeper/alias_functions.go

```diff
@@ -18,7 +17,7 @@ func (k Keeper) createAccountIfNotExists(ctx sdk.Context, user string) {
 	}
 	if !k.ak.HasAccount(ctx, addr) {
 		defer telemetry.IncrCounter(1, "new", "account")
-		k.ak.SetAccount(ctx, authtypes.NewBaseAccountWithAddress(addr))
+		k.ak.SetAccount(ctx, k.ak.NewAccountWithAddress(ctx, addr))
```

Comment:
`NewBaseAccountWithAddress` does not generate account number and sequence, it occurs the simulation tests error.

Reply:
Move to #1103